plugins/outputs/logzio/logzio.go

```diff
@@ -0,0 +1,185 @@
+package logzio
+
+import (
+	. "bytes"
+	"compress/gzip"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/internal/tls"
```

Comment:
Not valid package anymore (in master branch), should be change to `"github.com/influxdata/telegraf/plugins/common/tls"`